From 1fde981f30da369af36cd314249a02481600e55c Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Fri, 14 Jan 2011 09:03:16 -0500
Subject: [PATCH] python2.5: Do not use relative import *

relative imports are not yet fully functional on python2.5, thus it would lead to

Compiling /usr/lib/pymodules/python2.5/nipy/algorithms/registration/__init__.py ...
  File "/usr/lib/pymodules/python2.5/nipy/algorithms/registration/__init__.py", line 3
    from .resample import *
SyntaxError: 'import *' not allowed with 'from .'

Compiling /usr/lib/pymodules/python2.5/nipy/neurospin/group/tests/test_mixed_effects.py ...
  File "/usr/lib/pymodules/python2.5/nipy/neurospin/group/tests/test_mixed_effects.py", line 4
    from ..mixed_effects import *
SyntaxError: 'import *' not allowed with 'from .'

in this patch replacing with absolute path for .. and simply removing . for
current submodule imports
---
 nipy/algorithms/registration/__init__.py         |   10 +++++-----
 nipy/neurospin/group/tests/test_mixed_effects.py |    2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/nipy/algorithms/registration/__init__.py b/nipy/algorithms/registration/__init__.py
index 4eb4cb7..306df4b 100644
--- a/nipy/algorithms/registration/__init__.py
+++ b/nipy/algorithms/registration/__init__.py
@@ -1,12 +1,12 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
-from .resample import * 
-from .histogram_registration import * 
-from .affine import * 
-from .groupwise_registration import * 
+from resample import *
+from histogram_registration import *
+from affine import *
+from groupwise_registration import *
 
 from numpy.testing import Tester
 
 test = Tester().test
-bench = Tester().bench 
+bench = Tester().bench
 
diff --git a/nipy/neurospin/group/tests/test_mixed_effects.py b/nipy/neurospin/group/tests/test_mixed_effects.py
index cd52c13..065a0cc 100644
--- a/nipy/neurospin/group/tests/test_mixed_effects.py
+++ b/nipy/neurospin/group/tests/test_mixed_effects.py
@@ -1,7 +1,7 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 
-from ..mixed_effects import * 
+from nipy.neurospin.group.mixed_effects import *
 
 import numpy as np
 from numpy.testing import assert_array_almost_equal
-- 
1.7.2.3

